emperor/format.py

```diff
@@ -101,7 +101,7 @@ def format_pcoa_to_js(header, coords, eigvals, pct_var, custom_axes=[],
 
     # write the values for all the ellipses
     js_pcoa_string += '\nvar g_ellipsesDimensions = new Array();\n'
-    if coords_low != None and coords_high != None:
+    if is_empty(coords_low) and is_empty(coords_high):
```

Comment:
Is this equivalent? When `coords_low = None`, the previous code would result in a False but now it'd return True I think.